tests/integration/test_render.py

```diff
@@ -194,3 +199,36 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_hidden(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index in range(len(zones)):
+            z = '#zone-{}'.format(index + 1)
+            for s in self.SIDES:
```

Comment:
@agaylard Any specific reason why you're not using `selector` for `z` and `side` for `s` in these two lines?